configure.ac

```diff
@@ -483,6 +477,10 @@ AC_SUBST([PART_LDFLAGS])
 CFLAGS=$SAVE_CFLAGS
 LDFLAGS=$SAVE_LDFLAGS
 
+
+# For disk notification and fault LED.
+PKG_CHECK_MODULES([LIBLSM], [libstoragemgmt >= 1.3.0])
```

Comment:
If *lsm* is a hard requirement now, the ``--enable-lsm`` option doesn't make sense. Also, ``have_lsm`` is no longer being set -- how does the "old" code using it cope with that?